ts/Core/Series/Series3D.ts

```diff
@@ -67,7 +67,7 @@ class Series3D extends LineSeries {
      *
      * */
 
-    public static defaultOptions = merge(LineSeries.defaultOptions);
+    public static defaultOptions = merge(LineSeries.defaultOptions, {});
```

Comment:
Why the change? The `merge` function creates a deep copy when fed a single argument...

Reply:
This is to be consistent with the other change.